### ISSUE TITLE
[Bookstack with MariaDB] - Potential race condition when bookstack is bootstrapping.

### ISSUE THREAD

**Issue by larssb** [gabe565/charts#470]
### Chart Name

bookstack

### Chart Version

0.13.2

### Bug Description

When configuring

```yaml
mariadb:
  enabled: true
  auth:
```

A `MariaDB` database server is bootstrapped. However, `Bookstack` might come up before `MariaDB` is ready. Resulting in the database migrations/seeding script that creates proper tables and so on do not run.

Resulting in major errors when trying to access Bookstack. Of course as its entire data foundation layer is not ready yet.

---

## Workaround

- When MariaDB is fully up. Restart the Bookstack Pod >> it gets the connection to MariaDB and runs its database migrations successfully.

---

## Long term preferred fix

- That the Bookstack Pod stack continuously polls the MariaDB configured backend. For some acceptable max value. 5 minutes e.g. Keeps trying until max is hit ( e.g. 5 minutes ).

**Comment by gabe565** [comment]
You're right. I had originally seen that Bookstack waits for the database to be available so I didn't think this would be an issue, but it seems like the container's startup script fails, then it just continues to serve the site 🫤 It is fixed after a Bookstack pod restart, but that's not great UX.

I should be able to add an initContainer which waits for the database to go healthy to resolve this!
